components/table/ReportTable.tsx

```diff
@@ -372,9 +372,8 @@ function ReportTable({
     const reportsResponse =
         role === Role.ADMIN
             ? useTimesheetEntries(user, startDate, endDate)
-            : false
-    const allEntries =
-        reportsResponse && reportsResponse.isSuccess ? reportsResponse.data : []
+            : undefined
+    const allEntries = reportsResponse?.isSuccess ? reportsResponse.data : []
```

Comment:
`!!reportsResponse` might work here as well but it's up to you

Reply:
Field `reportsResponse.data` exists iff `reportsResponse.isSuccess === true`